cypress/integration/1-getting-started/e2e.spec.js

```diff
@@ -0,0 +1,114 @@
+describe("Open Page", () => {
+    beforeEach(() => {
+      cy.visit('./source/index.html');
+    });
+    // This test verifies the main page
+    it('loads and displays main', () => {
+      cy.get('.featured').should('be.visible');
+    });
+
+    // This test verifies the wrapper components
+    it('displays three recipe wrappers', () => {
+      cy.get('.recipes-wrapper').should('have.length', 3);
+      cy.get('.recipes-wrapper').each((element, index, list) => {
+          // expect(Cypress.$(element)).to.be.visible; // true when successfully retrieve carousels -> recipes from api
+          expect(index).to.be.greaterThan(-1);
+          expect(list).to.have.length(3);
+      });
+    });
+
+    // This test types in search bar a keyword and checks if top results are shown
+    it('searches keyword:tomato', () => {
+      cy.get('#heroSearch').type('tomato');
+      cy.get('#heroSearchButton').click();
+      cy.contains('h2', 'Top Results').should('be.visible');
+      cy.get('card-carousel', { timeout: 10000 }).should("be.visible");
+      // cy.get('card-carousel', { timeout: 10000 }).shadow().find('recipe-card').shadow().find('.title a').contains('tomato');
+    });
+    
+
+    // This test clicks a hyperlink in header and checks if it directs to the right page
+    it('displays correct subpage', () => {
+      // hover to Breakfast in Recipes dropdown
+      cy.get('#breakfastPage').invoke('show').click({ force: true });
+      //check to make sure page directs to the right section (breakfast)
+      cy.contains('h1', 'Breakfast Recipes').should('be.visible');
+      
+      // hover to Lunch in Recipes dropdown
+      cy.get('#lunchPage').invoke('show').click({ force: true });
+      //check to make sure page directs to the right section (lunch)
+      cy.contains('h1', 'Lunch Recipes').should('be.visible');
+      
+      // hover to Dinner in Recipes dropdown
+      cy.get('#dinnerPage').invoke('show').click({ force: true });
+      //check to make sure page directs to the right section (dinner)
+      cy.contains('h1', 'Dinner Recipes').should('be.visible');
+      
+      // hover to Dessert in Recipes dropdown
+      cy.get('#dessertPage').invoke('show').click({ force: true });
+      //check to make sure page directs to the right section (dessert)
+      cy.contains('h1', 'Dessert').should('be.visible');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group11/cse110-sp21-group11/pullRequest?prid=8548116)
```suggestion
      cy.contains("h1", 'Dessert').should('be.visible');
```